packages/next/src/server/base-server.ts

```diff
@@ -2688,7 +2716,8 @@ export default abstract class Server<
     }
 
     const didPostpone =
-      cacheEntry.value?.kind === 'PAGE' && !!cacheEntry.value.postponed
+      cacheEntry.value?.kind === 'PAGE' &&
+      typeof cacheEntry.value.postponed === 'string'
```

Comment:
not for this PR, but perhaps for the future: I think we should rename this value. seeing `postponed` makes me think I'm going to be working with a boolean, not postpone state.